gumby/modules/community_launcher.py

```diff
@@ -1,5 +1,14 @@
 from abc import ABCMeta, abstractmethod
 
+from Tribler.community.search.community import SearchCommunity
```

Comment:
This change should be reverted. This may look nicer, but moving the Tribler imports to the module level instead of the class level has two major downsides:

 1. Anyone who happens to be so unfortunate to have this module anywhere in their import chain, needs to have Tribler on their path. This may not seem like a big deal, until someone imports `custom_string from a_random_gumby_module` and has to figure out why he needs Tribler on the path.

 2. When running an experiment for _community X_ you should load the code from the _community X_ module. Now we may explicitly tell the launcher to only load _community X_, but we get the code from _community A_ through _community Z_ as well. Should these imports happen to conflict, the user won't know why: this is exactly what isolation is supposed to prevent.